test/Microsoft.ML.Tests/TrainerEstimators/MetalinearEstimators.cs

```diff
@@ -51,6 +51,36 @@ public void OVAUncalibrated()
             Done();
         }
 
+        /// <summary>
+        /// Test what OVA preserves key values for label.
+        /// </summary>
+        [Fact]
+        public void OvaKeyNames()
```

Comment:
>OvaKeyNames [](start = 20, length = 11)

Just curious, I heard something fairly troubling from @eerhardt, could we test this sort of scenario still works for things other than OVA?

Reply:
Any tests which use `TestEstimatorCore` will compare metadata from expected and resulted output schema.
I've change `AnnotationsForMulticlassScoreColumn` to always have TrainingLabelValue, and slotNames if key was text type, so technically we check that everywhere where we use `TestEstimatorCore`

---
In reply to: [269746627](https://github.com/dotnet/machinelearning/pull/3101#discussion_r269746627) [](ancestors = 269746627)